src/assets/stylesheets/Button.scss

```diff
@@ -240,35 +250,15 @@
   }
 
   .btn--primary {
-    $colours: $rpf-teal-400, $rpf-teal-200, $rpf-grey-200, $rpf-grey-700,
-      $rpf-teal-400, $rpf-teal-600;
+    $colours: var(--rpf-button-primary-background-color),
+      var(--rpf-button-primary-background-color-active),
+      var(--rpf-button-primary-background-color-disabled),
+      var(--rpf-button-primary-color-disabled),
+      var(--rpf-button-primary-color-disabled-focus),
+      var(--rpf-button-primary-background-color-hover);
     @include button-styling($colours...);
   }
 
-  .btn--secondary {
-    $colours: inherit, inherit, $rpf-grey-700, $rpf-grey-100, inherit, inherit,
-      $rpf-white;
-    @include button-styling($colours...);
-
-    border: 2px solid $rpf-teal-400;
-
-    &:active {
-      border: 2px solid $rpf-teal-900;
-    }
-
-    &:hover {
-      color: $rpf-grey-150;
-
-      svg {
-        fill: $rpf-grey-150;
-      }
-    }
-
-    &:focus-visible {
-      outline: 3px solid $rpf-brand-raspberry;
-    }
-  }
-
```

Comment:
Not sure I understand why this is being deleted but the equivalent rules for `primary` aren't 🤔

Reply:
I've removed all of this!

Reply:
Actually, don't worry on this one, I think I understand from what we've got further down in `index.scss`